packages/fast-components-react-base/src/slider-track-item/README.md

```diff
@@ -0,0 +1,24 @@
+## Slider track item
+
+A component designed to position itself at positions that correspond to a specific value on a *slider*'s track.  This can be used to create labels, highlighted regions or other decorations on a *slider*.  By default a *slider track item* renders above the *slider*'s front and back bars and below the thumbs.
```

Comment:
This first sentence reads a little weird to me. Maybe something more like:

"*Slide track items* are components positioned at specific values on a *slider*'s track to provide labels, highlight regions, and add other *slider* visual decorations."

Reply:
yep, better